be/src/exec/vectorized/arrow_to_json_converter.cpp

```diff
@@ -63,6 +63,31 @@ static enable_if_number<TypeClass, Status> convert_arrow_to_json_array(const Num
     M(Type::FLOAT)               \
     M(Type::DOUBLE)
 
+#define APPLY_FOR_ALL_TEMPORAL(M) \
```

Comment:
keep the code style consistent with this file

TYPE_GUARD(PhysicalTemporalTypeGuard, is_physical_temporal_type, Type::DATE32, Type::DATE64,Type::TIME32, Type::TIME64,Type::TIMESTAMP);

Reply:
The `TYPE_GUARD` and `VALUE_GUARD` is not appropriate to this case. 
- I need a plain function to check whether the type is temporal
- The `VALUE_GUARD` generates a template function, which needs template dispatch